packages/component-library/package.json

```diff
@@ -39,6 +39,15 @@
   "scripts": {
     "prepublish": "tsc --project tsconfig.dist.json",
     "build": "yarn prepublish",
+    "test-scss-compile": "webpack build --config ./styles/__test/webpack.config.js",
     "clean": "rimraf -g '**/*.js.map' 'index.js*' 'index.d.ts' 'components/**/*.js' 'components/**/*.js.map' 'components/**/*.d.ts' 'util/**/*.js' 'util/**/*.js.map' 'util/**/*.d.ts'"
+  },
+  "devDependencies": {
+    "sass": "^1.62.1",
+    "sass-loader": "^13.2.2",
+    "ts-loader": "^9.4.2",
+    "typescript": "^5.0.4",
+    "webpack": "^5.83.1",
+    "webpack-cli": "^5.1.1"
   }
```

Comment:
This is quite gross :( Aren't these all installed in the root package.json already?

I don't think having two versions of `typescript` in the repo is good either. Somehow I can see things clashing :'(

Reply:
Yeah good catch, I forgot these are in the root package for Storybook, assumed Storybook kept these deps to itself lol